docs/source/trainer/using_the_trainer.rst

```diff
@@ -271,6 +271,25 @@ engineering. We currently support the ``cpu`` and ``gpu`` devices.
         device='cpu'
     )
 
+TRaining on TPU
+~~~~~~~~~~~~~~~
+
+Now you can also use composer to train your model on single core ``tpus``
+in ``bf16`` precision. You will need to have ``torch_xla`` installed using
+instructions here https://github.com/pytorch/xla.
+
+.. testcode::
+
+    from composer import Trainer
+
+    trainer = Trainer(
+        model=model,
+        train_dataloader=train_dataloader,
+        eval_dataloader=eval_dataloader,
+	max_duration='2ep',
+	device='tpu'
+    )
+    
```

Comment:
can you add a `.. note::` here that we will add multi-core support in future releases?